src/libopensc/cardctl.h

```diff
@@ -1001,10 +1023,13 @@ typedef struct sc_cardctl_openpgp_keystore_info {
 			size_t n_len;
 		} rsa;
 		struct {
+			u8 keyformat;	/* SC_OPENPGP_KEYFORMAT_EC_... */
```

Comment:
clang-format suggested the following change:


```
-			u8 keyformat;	/* SC_OPENPGP_KEYFORMAT_EC_... */
+			u8 keyformat; /* SC_OPENPGP_KEYFORMAT_EC_... */
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/cardctl.h -->